Code/Framework/AzNetworking/AzNetworking/UdpTransport/UdpNetworkInterface.h

```diff
@@ -55,8 +55,16 @@ namespace AzNetworking
     //! ### Fragmentation
     //! 
     //! If the raw packet size exceeds the configured maximum transmission unit (MTU) then the packet is broken into
-    //! multiple reliable fragments to avoid fragmentation at the routing level. Fragments are always reliable so the original
-    //! packet can be reconstructed. Operations that alter the payload generally follow this step so that they can be
+    //! multiple fragments to avoid fragmentation at the routing level. Fragment reliability is controlled by the cvar
+    //! net_FragmentsAlwaysReliable. When true, fragments are always reliable to ensure the source packet can be reconstructed.
```

Comment:
Maybe something more like:

```
   //! When true, fragments are always sent reliably to ensure the source packet can be reconstructed.
    //! This means unreliable packets exceeding MTU will be split into several reliable packets to guarantee delivery but at a cost of extra network processing.
    //! (Default) When false, fragments inherit the reliability type of their source packet. This may cause non-reliable packet transmission to fail if packet is fragmented, 
    //! if even one fragment fails to be delivered due to network conditions.
```